app/frontend/src/features/profile/view/About.test.tsx

```diff
@@ -0,0 +1,36 @@
+import { render, screen } from "@testing-library/react";
+import { User } from "pb/api_pb";
+import { service } from "service";
+import wrapper from "test/hookWrapper";
+import { getLanguages, getRegions } from "test/serviceMockDefaults";
+
+import About from "./About";
+
+const getLanguagesMock = service.resources.getLanguages as jest.MockedFunction<
+  typeof service.resources.getLanguages
+>;
+
+const getRegionsMock = service.resources.getRegions as jest.MockedFunction<
+  typeof service.resources.getRegions
+>;
+
+beforeEach(() => {
+  getLanguagesMock.mockImplementation(getLanguages);
+  getRegionsMock.mockImplementation(getRegions);
+});
+
+function renderAbout(user?: User.AsObject) {
+  render(<About user={user || defaultUser} />, { wrapper });
+}
+describe("About (user)", () => {
+  it("displays both visited and lived regions", async () => {
+    renderAbout();
+
+    // Australia is displayed both for lived and visited regions
+    expect((await screen.findAllByText("Australia")).length).toBe(2);
+
+    screen.getByText("Sweden");
+    screen.getByText("Finland");
+    screen.getByText("United States");
```

Comment:
```suggestion
    expect(screen.getByText("Sweden")).toBeVisibie();
    expect(screen.getByText("Finland")).toBeVisibie();
    expect(screen.getByText("United States")).toBeVisibie();
```
Use assertions from jest dom for clearer error messages when the test fails

Reply:
`toBeVisible` will never be called if `getByText` already throws. So also here, the error message thrown will be identical as in my version without `expect(...).toBeVisible()`

Reply:
I feel like this was addressed in another comment in a different PR, so fair point on same error on `getByText` on its own. So this probably comes back to the test style nitpick - but I think to make the intent clearer that it's a result we care about, I'd prefer we explicitly make an assertion rather than doing `screen.*` on its own, as that _could_ be something we write when we want a handle to await on (e.g.for a mocked API to resolve), but not necessarily care about the result.